### DISCUSSION TITLE
mpich configuration settings for single macine run.

### DISCUSSION THREAD

**Discussion by tarunkumar987** [pmodels/mpich#7063] [General]
Our usecase is utililizing multiple cores on a single machine. We recenly faced following assertion for a single machine run and crash is at  a place which is utilizing a TCP handshake. My queston is why TCP is being used for a single machine run. Is there a better device/protocol we can use for the use case. Is there a configuration option we should use?

FYR
https://github.com/pmodels/mpich/discussions/7052

Thanks

**Comment by tarunkumar987** [comment]
Any update here? We are stuck at our end here in the mentioned crash https://github.com/pmodels/mpich/discussions/7052

**Comment by hzhou** [comment]
Without knowing the nature of the security app you mentioned, we have no clue on the actual cause and no idea on what to fix. Did you try setting the loopback iface as suggested in https://github.com/pmodels/mpich/discussions/7052#discussioncomment-10077443?
